Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/sdk/common/azure/serverless/AzureSparkServerlessAccount.java

```diff
@@ -104,6 +109,34 @@ public URI getUri() {
         return uri;
     }
 
+    @Nullable
+    public URI getJobManagementURI() {
+        if (getId() == null) {
+            return null;
+        }
+
+        try {
+            String url = AuthMethodManager.getInstance().getAzureManager().getPortalUrl()
```

Comment:
`AuthMethodManager.getInstance().getAzureManager()` may return `null` for non-signed-in user